build.gradle

```diff
@@ -123,21 +140,92 @@ configure(subprojects.findAll {it.name != 'sda-commons-bom' && it.name != 'sda-c
  * - example modules and dependency-check should not be published at all
  */
 List<String> unpublishedModules = Arrays.asList("sda-commons-bom", "sda-commons-dependencies", "sda-commons-dependency-check")
-configure(subprojects.findAll {
-    !unpublishedModules.contains(it.name) && !it.name.endsWith("-example")
-}) {
+configure(subprojects.findAll { !unpublishedModules.contains(it.name) && !it.name.endsWith("-example") }) {
```

Comment:
The modules `sda-commons-bom` and `sda-commons-dependencies` are not uploaded. So the version in teh snapshot repository at sonatype are not usable yet.

They are configured in their own gradle files: https://github.com/SDA-SE/sda-dropwizard-commons/blob/master/sda-commons-bom/build.gradle

We can maybe check if we can also configure them in the main gradle file.